build.gradle

```diff
@@ -195,3 +197,16 @@ compileTestKotlin {
 }
 // However if you are in a multi-project build, dev time needs unobfed jar files, so you can delay the obfuscation until publishing by doing
 // publish.dependsOn('reobfJar')
+modrinth {
+    token = System.getenv("MODRINTH_TOKEN")
+    projectId = "iTyN4Kv7"
+    versionNumber = "$project.version"
+    versionName = "eln2-$project.version"
+    versionType = "alpha"
+    uploadFile = shadowJar
+    gameVersions = [ "$mcVersion" ]
+    changelog = file('changes.md').text
+    dependencies {
+        required.project("ordsPcFz")
```

Comment:
Can we leave a comment here that it depends on KFF? Then I don't need to look up the ID.